quickwit/quickwit-search/src/service.rs

```diff
@@ -303,6 +303,8 @@ pub struct SearcherContext {
     pub split_stream_semaphore: Semaphore,
     /// Recent sub-query cache.
     pub leaf_search_cache: LeafSearchCache,
+    /// Query string max length.
+    pub query_string_max_length: usize,
```

Comment:
```suggestion
```

Reply:
it is in the searcher_config, which is already in the SearcherContext.

Reply:
fix
https://github.com/quickwit-oss/quickwit/pull/3566/commits/9d02896663ee4181c52422baf6c61cf55b4b29f5